compilation crashes for GE21 systems

<!--- Provide a general summary of the issue in the Title above -->
## Brief summary of issue
<!--- Provide a description of the issue, including any other issues or pull requests it references -->
When setting the `GEM_VARIANT` to ge21 the compilation crashes. For ge11 code compiles
### Types of issue
<!--- Propsed labels (see CONTRIBUTING.md) to help maintainers label your issue: -->
- [x] Bug report (report an issue with the code)
- [ ] Feature request (request for change which adds functionality)

## Expected Behavior
<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->
Should compile
## Current Behavior
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->
```bash
make
***
/home/mdalchen/work/cms-gem-daq-project/ctp7_modules/src/amc/sca.cpp: In function ‘void readAllSCAADCSensors(const wisc::RPCMsg*, wisc::RPCMsg*)’:
/home/mdalchen/work/cms-gem-daq-project/ctp7_modules/src/amc/sca.cpp:392:95: error: ‘FULL_OH_MASK’ is not a member of ‘amc’
   const uint32_t ohMask   = request->get_key_exists("ohMask") ? request->get_word("ohMask") : amc::FULL_OH_MASK;
                                                                                               ^
/home/mdalchen/work/cms-gem-daq-project/ctp7_modules/src/amc/sca.cpp:392:95: note: suggested alternative:
In file included from /home/mdalchen/work/cms-gem-daq-project/ctp7_modules/src/amc/sca.cpp:7:0:
/home/mdalchen/work/cms-gem-daq-project/ctp7_modules/include/hw_constants.h:23:28: note:   ‘amc::ge11::FULL_OH_MASK’
         constexpr uint32_t FULL_OH_MASK = 0xfff; ///< Reading of all OptoHybrids are enabled.
                            ^
make[1]: *** [/home/mdalchen/work/cms-gem-daq-project/ctp7_modules/src/linux/arm/amc/sca.o] Error 1
make[1]: Leaving directory `/home/mdalchen/work/cms-gem-daq-project/ctp7_modules'
make: *** [amc] Error 2
```
### Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
Attempt to compile

## Possible Solution (for bugs)
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->
```cpp
diff --git a/include/hw_constants.h b/include/hw_constants.h
index 0533286..84c3206 100644
--- a/include/hw_constants.h
+++ b/include/hw_constants.h
@@ -27,6 +27,7 @@ namespace amc {
      */
     namespace ge21 {
         constexpr uint32_t OH_PER_AMC = 12;    ///< The number of OptoHybrids per AMC.
+        constexpr uint32_t FULL_OH_MASK = 0xfff; ///< Reading of all OptoHybrids are enabled.
     }

     using namespace GEM_VARIANT;
```
## Context (for feature requests)
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: release/legacy-1.1
* Shell used:

<!--- Template thanks to https://www.talater.com/open-source-templates/#/page/98 -->

--- a/Makefile
+++ b/Makefile
@@ -193,4 +193,6 @@ clean: cleanrpm
 cleandoc:
 	@echo "TO DO"
 
+BUILD_VERSION:=$(BUILD_VERSION).$(GEM_VARIANT)
+
 -include $(Dependencies)

--- a/include/hw_constants.h
+++ b/include/hw_constants.h
@@ -27,6 +27,7 @@ namespace amc {
      */
     namespace ge21 {
         constexpr uint32_t OH_PER_AMC = 12;    ///< The number of OptoHybrids per AMC.
+        constexpr uint32_t FULL_OH_MASK = 0xfff; ///< Reading of all OptoHybrids are enabled.
     }
 
     using namespace GEM_VARIANT;

